files/default/sql/ddl/3.8.0__initial_tables.sql

```diff
@@ -275,6 +275,7 @@ CREATE TABLE `executions` (
                               `execution_stop` bigint(20) DEFAULT NULL,
                               `stdout_path` varchar(255) COLLATE latin1_general_cs DEFAULT NULL,
                               `stderr_path` varchar(255) COLLATE latin1_general_cs DEFAULT NULL,
+                              `notebook_out_path` varchar(255) COLLATE latin1_general_cs DEFAULT NULL,
```

Comment:
Also need to be added here: https://github.com/logicalclocks/hopsworks-chef/blob/master/files/default/sql/ddl/updates/3.8.0.sql. ```
ALTER TABLE `hopsworks`.`executions`
    ADD COLUMN `notebook_out_path` varchar(255) COLLATE latin1_general_cs DEFAULT NULL;
```